### PR TITLE
add a simple flow

### PR DESCRIPTION
**Pull request by xczhai**
just add a png to describe.
**A picture is worth a thousand words**

### CODE REVIEW

File: src/docs/openvino_genai.png
Comment (Wovchena) [inline src/docs/openvino_genai.png:1]: Can you try an image with alpha channel to preserve github night theme?

Comment (xczhai) [reply src/docs/openvino_genai.png:1]: The fonts is black. White background should be kept, I think.

Comment (Wovchena) [reply src/docs/openvino_genai.png:1]: Adding bubbles around the text similar to the one you have around OpenVINO GenAI in the middle would resolve that.

File: src/docs/openvino_genai.jpg
Comment (yury-gorbachev) [inline src/docs/openvino_genai.jpg:1]: I propose to remove path 3 as it is quite confusing for a person to follow (how to get IR?) and add Model Scope as a path similar to Hugging Face instead.
We currently don't document this well (WIP), but one can download model from MS and use optimum-cli export to convert model to IR. 
cc: @eaidova

Comment (yury-gorbachev) [reply src/docs/openvino_genai.jpg:1]: Another aspect - how we can alter this picture later? do you have a source that you can put somewhere for later edits or should we redo image later again?

Comment (ilya-lavrenov) [reply src/docs/openvino_genai.jpg:1]: > how we can alter this picture later?

better to use SVG then

**Comment by ilya-lavrenov** [comment]
@xczhai ping

**Comment by xczhai** [comment]
> @xczhai ping

Hi @ilya-lavrenov @yury-gorbachev 
sorry for reply late because of focusing on customer's tasks.

1. remove previous path 3
2. upload svg, which can be edited.

**Review by yury-gorbachev — APPROVED** [review]

**Pull request merged**